convert_tendl.py

```diff
@@ -63,6 +63,14 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
         'metastables': os.path.join(ace_files_dir, 'ace-17', '*m'),
         'compressed_file_size': '2.1 GB',
         'uncompressed_file_size': '14 GB'
+    },
+    '2019': {
+        'base_url': 'https://tendl.web.psi.ch/tendl_2019/tar_files/',
+        'files': ['tendl19c.tar.bz2'],
+        'neutron_files': os.path.join(ace_files_dir, 'tendl19c', '*'),
+        'metastables': os.path.join(ace_files_dir, 'tendl19c', '*m'),
+        'compressed_file_size': '2.2 GB',
+        'uncompressed_file_size': '6.8 GB'
```

Comment:
Using the terminal command "du -h" I get a larger file size here
2.4G	./tendl-2019-hdf5
7.7G	./tendl-2019-ace